tests/suites/manual/deploy_manual.sh

```diff
@@ -10,10 +10,10 @@ test_deploy_manual() {
 		cd .. || exit
 
 		case "${BOOTSTRAP_PROVIDER:-}" in
-		"lxd" | "lxd-remote" | "localhost")
+		"lxd")
 			run "run_deploy_manual_lxd"
 			;;
-		"aws" | "ec2")
```

Comment:
This seems to have been changed to `ec2` everywhere else...

Reply:
https://github.com/juju/juju/pull/17448/files#diff-022e60c5ac1d2ca17b86164b81b834a6b30c57a7e47950e26a9bc3a463fc3ec5L16

You'll have to ask Harry

Reply:
But agreed, this looks wrong